src/Makefile

```diff
@@ -171,7 +171,7 @@ cudamatrix: base util matrix
 nnet: base util hmm tree matrix cudamatrix
 nnet2: base util matrix lat gmm hmm tree transform cudamatrix
 nnet3: base util matrix lat gmm hmm tree transform cudamatrix chain fstext
-rnnlm: base util matrix cudamatrix nnet3 lm hmm
+rnnlm: base util matrix cudamatrix nnet3 chain lm hmm
```

Comment:
what linking error was this intended to solve?

Reply:
sorry, it was caused by a problem in my environment. removed this.